tools/azure-sdk-tools/gh_tools/vnext_issue_creator.py

```diff
@@ -56,6 +60,45 @@ def get_build_link(check_type: CHECK_TYPE) -> str:
         f"https://dev.azure.com/azure-sdk/internal/_build/results?buildId={build_id}&view=logs&j={job_id}&t={next_id}"
     )
 
+def get_build_info(build_link: str, check_type: CHECK_TYPE, service_directory: str, package_name: str) -> str:
+    """Get the build info from the build link."""
+    build_id = os.getenv("BUILD_BUILDID")
+    job_id = os.getenv("SYSTEM_JOBID")
+    timeline_link =f"https://dev.azure.com/azure-sdk/internal/_apis/build/builds/{build_id}/timeline?api-version=6.0"
+
+    if not in_ci():
+        DEVOPS_RESOURCE_UUID = "499b84ac-1321-427f-aa17-267ca6975798"
+        token = DefaultAzureCredential().get_token(f"{DEVOPS_RESOURCE_UUID}/.default").token
+    else:
+        token = os.environ["SYSTEM_ACCESSTOKEN"]
+    AUTH_HEADERS = {"Authorization": f"Bearer {token}"}
+
+    # Make the API request
+    response = requests.get(timeline_link, headers=AUTH_HEADERS)
+    logging.info(f"Response: {response.text}")
+    response_json = json.loads(response.text)
+    # return response_json
+    response_two = []
+    add = False
+    next_id = "b33d1587-3539-5735-af43-e3e62f02ca4b"
+    import gzip
+    import io
+    try:
+        for task in response_json["records"]:
+            if "Run Pylint Next" in task["name"]:
+                log_link = task['log']['url'] + "?api-version=7.1"
+                # Get the log file from the build link
+                log_output = requests.get(log_link, headers=AUTH_HEADERS)
+                build_output = log_output.content.decode("utf-8")
+                new_output = (build_output.split(f"next-pylint: commands[3]> python /mnt/vss/_work/1/s/eng/tox/run_pylint.py -t /mnt/vss/_work/1/s/sdk/{service_directory}/{package_name} --next=True")[1]).split(f"ERROR:root:{package_name} exited with linting error")[0]
```

Comment:
should we assume that this step always runs in ubuntu? I think we need to double-check

Reply:
![image](https://github.com/user-attachments/assets/b0a5cd05-4f87-4264-ad5b-7347694237df)
pool is set to linux